confluent_kafka/avro/AvroProducer.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+#
+# Copyright 2016 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import logging
+
+from confluent_kafka.avro.serializer import SerializerError
+
+
+class AvroProducer(object):
+    '''
+        Kafka Producer client which does avro schema encoding to messages.
+        Handles schema registration, Message serialization.
+
+        Constructor takes below parameters
+
+        @:param: producer: confluent_kafka.Producer object
+        @:param: message_serializer: Message Serializer object
+    '''
+    def __init__(self, producer, message_serializer):  # real signature unknown; restored from __doc__
+        self.producer = producer
+        self.serializer = message_serializer
+        self.log = logging.getLogger(__name__)
+
+    def produce(self, topic, value=None, value_avro_schema=None, key=None, key_avro_schema=None, *args, **kwargs):
+        '''
+            Sends message to kafka by encoding with specified avro schema
+            @:param: topic: topic name
+            @:param: value: A dictionary object
+            @:param: value_avro_schema : Avro schema for value
```

Comment:
I'd shorten these names to `value_schema` and `key_schema` -- we're in an Avro-specific class, so they couldn't really be anything else.

I don't know if people might also find it useful to be able to set the schema as a default in the constructor? In languages like Java where the schema is attached anyway and is handled automatically if you use code generation, this isn't as annoying. It might be a bit more annoying to have to pass the same schema (the normal case) into every produce call. Not sure how valuable this will be since usually you only have a small number of call sites to `produce()` anyway, but seems annoying to have to carry the schema around separately everywhere.